iwyu_include_picker.cc

```diff
@@ -473,9 +473,11 @@ const IncludeMapEntry stdlib_c_include_map[] = {
   // mapping.)  Here is how I identified the files to map:
   // $ for i in /usr/include/c++/4.4/c* ; do ls /usr/include/`basename $i | cut -b2-`.h /usr/lib/gcc/*/4.4/include/`basename $i | cut -b2-`.h 2>/dev/null ; done
   //
-  // These headers are defined in C++14 [headers]p3.  You can get them with
-  // $ sed -n '/begin{floattable}.*{tab:cpp.c.headers}/,/end{floattable}/p' lib-intro.tex | grep tcode | perl -nle 'm/tcode{<c(.*)>}/ && print qq@  { "<$1.h>", kPublic, "<c$1>", kPublic },@' | sort
-  // on https://github.com/cplusplus/draft/blob/master/source/lib-intro.tex
+  // These headers are defined in [headers.cpp.c].
+  // https://github.com/cplusplus/draft/blob/c+%2B20/source/lib-intro.tex
+  //
+  // $ wget https://raw.githubusercontent.com/cplusplus/draft/c%2B%2B20/source/lib-intro.tex
```

Comment:
Nice! Could you fold this into a one-liner so the instruction doesn't require disk for lib-intro.tex? E.g.
```
$ curl https://raw.githubusercontent.com/cplusplus/draft/c%2B%2B20/source/lib-intro.tex | sed -n '/begin{multicolfloattable}.*{headers.cpp.c}/,/end{multicolfloattable}/p' ...
```